crates/fdev/src/build.rs

```diff
@@ -21,6 +21,21 @@ pub(crate) use contract::*;
 const DEFAULT_OUTPUT_NAME: &str = "contract-state";
 const WASM_TARGET: &str = "wasm32-unknown-unknown";
 
+#[cfg(windows)]
+pub const NPM_BUILD_COMMAND: &'static str = "npm.cmd";
+#[cfg(windows)]
+pub const TSC_BUILD_COMMAND: &'static str = "tsc.cmd";
+#[cfg(windows)]
+pub const WEBPACK_BUILD_COMMAND: &'static str = "tsc.cmd";
+
+
+#[cfg(not(windows))]
+pub const NPM_BUILD_COMMAND: &'static str = "npm";
+#[cfg(not(windows))]
+pub const TSC_BUILD_COMMAND: &'static str = "tsc";
+#[cfg(not(windows))]
+pub const WEBPACK_BUILD_COMMAND: &'static str = "tsc.cmd";
```

Comment:
this one should be `tsc` no?

Reply:
In Linux yes. In windows no. Could not compile on windows else.

Reply:
wait... yes should be TSC_BUILD_COMMAND yes...

Reply:
rebase issue